azurerm/resource_arm_traffic_manager_endpoint.go

```diff
@@ -281,5 +365,46 @@ func getArmTrafficManagerEndpointProperties(d *schema.ResourceData) *trafficmana
 		endpointProps.MinChildEndpoints = &mci64
 	}
 
+	templist := d.Get("subnet").([]interface{})
```

Comment:
Could we apply the recommendations for the header variables here too?

Reply:
Yes